### PR TITLE
Added the Java Code for an Auto Service app

### PR DESCRIPTION
**Pull request by VadimDonic**
Was created 4 different Java classes: Car, Tool, Worker, Garage and additional one - ManageAutoService class. I used different types of constructors with each objective which i previously made.

### CODE REVIEW

File: src/main/java/com/donic/classesandobjects/autoservicetask/ManageAutoService.java [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+        System.out.println(" Automobil cu VinCod: " + audiA5Quattro.vinNumber + " cu cutia " + audiA5Quattro.gearType +
+                " cu pretul " + audiA5Quattro.carPrice + " care consuma " + audiA5Quattro.fuelType + " anul producerii " +
+                audiA5Quattro.productionYear + " model " + audiA5Quattro.model + " marca " + audiA5Quattro.make);
+
+
+        Tool ciocan = new Tool("Ciocan");
+        ciocan.uniqueCode = 123124;
+        System.out.println(" Numele si numarul de instrument este " + ciocan.name + " cu numarul uniq " + ciocan.uniqueCode);
+
+        Tool Elevator = new Tool();
```
Comment (iulik98) [inline src/main/java/com/donic/classesandobjects/autoservicetask/ManageAutoService.java]: Variable name should start with lower case.

Comment (AndreiSecuQA) [reply src/main/java/com/donic/classesandobjects/autoservicetask/ManageAutoService.java]: Could you please use git commit amend and change the variable name as per the syntax rules?

Comment (VadimDonic) [reply src/main/java/com/donic/classesandobjects/autoservicetask/ManageAutoService.java]: > Could you please use git commit amend and change the variable name as per the syntax rules?

Sry , I’m working right now (full time job).

I can fix my mistake only after 18:30

**Review by iulik98 — APPROVED** [review]
Good Job!

**Review by CiobanuVladimir — APPROVED** [review]
Great job

**Pull request merged**